sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaFormat.cs

```diff
@@ -1,15 +1,44 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using Azure.Core;
+using System;
+using System.ComponentModel;
 
 namespace Azure.Data.SchemaRegistry
 {
-    /// <summary>
-    /// The format used when serializing SchemaRegistry messages.
-    /// </summary>
-    [CodeGenModel("SerializationType")]
-    public readonly partial struct SchemaFormat
+    /// <summary> The SerializationType. </summary>
+    public readonly partial struct SchemaFormat : IEquatable<SchemaFormat>
     {
+        private readonly string _value;
+
+        /// <summary> Initializes a new instance of <see cref="SchemaFormat"/>. </summary>
+        /// <exception cref="ArgumentNullException"> <paramref name="value"/> is null. </exception>
+        public SchemaFormat(string value)
+        {
+            _value = value ?? throw new ArgumentNullException(nameof(value));
+        }
+
+        private const string AvroValue = "avro";
+
+        /// <summary> Avro Serialization schema type. </summary>
+        public static SchemaFormat Avro { get; } = new SchemaFormat(AvroValue);
```

Comment:
just wanted to confirm that we accept case insensitive Avro in registerSchema/getSchemaProperties and set SchemaProperties.format = "Avro" when we return SchemaProperties?

Reply:
No, it just always uses "avro". What do you mean when you say the service returns "Avro"? I thought it was just part of the content-type as serialization=avro.

Reply:
It shouldn't make a difference here as we use case-insensitive comparison for the SchemaFormat type.

Reply:
Regenerated with "Avro"